### DISCUSSION TITLE
MKV videos does not work on iOS

### DISCUSSION THREAD

**Discussion by ShahmeerShahid** [KiraLT/torrent-stream-server#245] [Q&A] [answered]
When opening a stream on IOS, I get the following error:

![File_000](https://user-images.githubusercontent.com/37679657/127759657-7c09e88b-e289-4de0-ae97-9462df5b7df3.png)

The same page/stream works on OSX and Android.

**Comment by KiraLT** [comment, marked as answer]
The issue is because the browser doesn't support MKV files. Some browsers have experimental support (e.g. chrome, does not apply to iOS chrome), but they have limitations (like #16).

To view MKV videos on devices that don't support MKV, you need a browser extension or open the video with an external player (like VLC). Most external players (on both PC and mobile) and play video from URL, so you just need copy stream URL.

Also, #181 could help - in theory, it is possible to re-encode video to browser-friendly format like MP4.
